docs/content/preview/contribute/core-database/build-from-src-macos.md

```diff
@@ -93,11 +93,17 @@ echo 'export PATH="/opt/local/homebrew/opt/openjdk@11/bin:$PATH"' >>~/.bashrc
 echo 'export PATH="/usr/local/opt/openjdk@11/bin:$PATH"' >>~/.bashrc
 ```
 
+### yugabyted-ui
+
+{{% readfile "includes/yugabyted-ui.md" %}}
+
 ## Build the code
 
 {{% readfile "includes/build-the-code.md" %}}
 
-### Build release package
+### Build release package (optional)
+
```

Comment:
Include an introduction statement such as "Perform the following steps to build the release package.".

Also, include the steps as bullets: (yugabyted-ui requirement, Install the following additional packages:, Run the yb_release script to build a release package)

Reply:
Leaving this also for followup.